projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.style.less

```diff
@@ -0,0 +1,21 @@
+@import 'taiga-ui-local';
+
+.table-caption {
+    text-align: left;
+    margin-bottom: 0.5rem;
+}
+
+.copy {
+    .fullsize();
+    .transition(opacity);
+    opacity: 0;
+
+    ::ng-deep button {
```

Comment:
doesn't appearance="table" fits this case?

Reply:
:(

https://github.com/Tinkoff/taiga-ui/blob/943c57c2103cde34ab1a5da8f6c9b8551f78b4bf/projects/addon-doc/src/components/copy/copy.template.html#L5

Reply:
got it :(